src/main/kotlin/no/nav/helsearbeidsgiver/pdl/domene/DomeneIntern.kt

```diff
@@ -0,0 +1,30 @@
+@file:UseSerializers(LocalDateSerializer::class)
+
+package no.nav.helsearbeidsgiver.pdl.domene
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.UseSerializers
+import no.nav.helsearbeidsgiver.utils.json.serializer.LocalDateSerializer
+import java.time.LocalDate
+
+@Serializable
+data class FullPerson(
```

Comment:
PDL eksponerer skjema. Er det noe grunn å ikke bruke skjemaen ?

Reply:
Hvis du med skjema mener PDLs egne klasser, så tenker jeg at de er unødvendig kompliserte og utfyllende for vår bruk.